### DISCUSSION TITLE
Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by espinuevajaime** [community/community#112392] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I have received an error when I subscribed to github copilot. I have tried logging out and re logging in, reinstall copilot extension and I am still getting the Invalid copilot token: missing token: 403. 

```
2024-03-14 10:15:07.824 [info] [auth] Invalid copilot token: missing token: 403 
2024-03-14 10:15:07.850 [error] [default] Extension activation failed: "Contact Support. You are currently logged in as espinuevajaime." 
```

**Comment by xuanyanwow** [comment]
+1

**Comment by tw4test** [comment]
same here, there were no problems before yesterday

**Comment by xuanyanwow** [comment]
same question

**Comment by manyuan-li** [comment]
sam here, I can use coplit yesterday

**Comment by jianzhihuang** [comment]
+1

**Comment by liasica** [comment]
me too

**Comment by Hwenyi** [comment]
me too

**Comment by ghost** [comment]
me too

**Comment by huangzh278** [comment]
me too +1

**Comment by StivenYang** [comment]
I don’t know why this problem occurs and needs to be solved

![image](https://github.com/community/community/assets/20460759/dd3f54d8-dd31-4a2d-8591-053d24c4b364)

**Reply by StivenYang** [reply]
my copilot had worked. this problem had fixed. thank you!

**Reply by manyuan-li** [reply]
What did you do to make it works? thank u

**Reply by StivenYang** [reply]
It happened again. I submitted a work order to Github

**Comment by jtsang4** [comment]
Meet the same question.

**Comment by HydroCarbon** [comment]
+1

**Comment by davecheney** [comment, marked as answer]
Please contact support, we cannot assist you via this forum in this matter. Thank you.

**Reply by davecheney** [reply]
We are unable to assist you via this forum. Please contact support.

**Reply by jianzhihuang** [reply]
If posting feedback directly on the forum isn't an option, what kind of content can be shared there?

**Reply by ligaofeng0901** [reply]
There is no support!!!

**Reply by hsenag** [reply]
The error indicates an issue with your account, not a technical issue with Copilot. The details will be private to you so we would not post them in a public forum.

**Comment by ligaofeng0901** [comment]
+1

**Comment by ligaofeng0901** [comment]
please fix this problem

**Comment by hqzh** [comment]
+1

**Comment by nimrahSohail** [comment]
> ### Select Topic Area
> Bug
> 
> ### Body
> I have received an error when I subscribed to github copilot. I have tried logging out and re logging in, reinstall copilot extension and I am still getting the Invalid copilot token: missing token: 403.
> 
> ```
> 2024-03-14 10:15:07.824 [info] [auth] Invalid copilot token: missing token: 403 
> 2024-03-14 10:15:07.850 [error] [default] Extension activation failed: "Contact Support. You are currently logged in as espinuevajaime." 
> ```

I just heard about copilot and got confused it's not working for me. I am using the github developer pack and facing the same error

**Reply by nimrahSohail** [reply]
I figured out the issue and it's resolved now. 
I just sign out and then sign in again using github account in vscode and yayy copilot works pretty fine.

**Reply by duckthihn** [reply]
I'm having the same problem as you and just resolved it by signing out and then signing in. However, that error will happen again and I have to repeat signing out and in again :(.

**Comment by gelove** [comment]
same question.
I have education pack.

**Comment by xjh2000** [comment]
same question

**Comment by alalshow** [comment]
해결하신분있나요? 미치겠네요.
